Dockerfile

```diff
@@ -17,7 +18,13 @@ RUN if [ -z "${GO_VERSION_ARG}" ]; then \
       GO_VERSION=${GO_VERSION_ARG}; \
     fi; \
     rm -rf /usr/local/go; \
-    curl -L --output - "https://golang.org/dl/go${GO_VERSION}.linux-${GO_PLATFORM}.tar.gz" | tar -xz -C /usr/local/
+    curl -L --output - "https://golang.org/dl/go${GO_VERSION}.linux-${GO_PLATFORM}.tar.gz" | tar -xz -C /usr/local/; \
+    mkdir -p opt/ol; \
+    curl -L -o opt/ol/wlp.zip "https://repo1.maven.org/maven2/io/openliberty/openliberty-kernel/${LIBERTY_VERSION}/openliberty-kernel-${LIBERTY_VERSION}.zip"; \
```

Comment:
Although both WLO and OLO will be using Open Liberty, suggest using `liberty` instead of `opt/ol/wlp` as the directory to abstract out which liberty is used